elixir/apps/web/lib/web/live/resources/edit.ex

```diff
@@ -72,7 +72,7 @@ defmodule Web.Resources.Edit do
               </p>
             </div>
 
-            <.filters_form :if={@traffic_filters_enabled?} form={@form[:filters]} />
```

Comment:
Not sure -- I suppose it's a billing flag now?

Reply:
That's not something that would ever happen through the config though right?

If we're planning to roll this out to everyone right now (and just gray it out for free tier users), then I would say we probably are done with the actual [feature flag](https://github.com/firezone/firezone/blob/b45a302749df2b34acb6eca0de1a2c13b9adb64b/elixir/apps/domain/lib/domain/config/definitions.ex#L641).

That said, if we wanted to wait on removing the flag definition just in case we need to "rollback" the feature, we can just create an issue to make sure we don't forget about it.

Reply:
I see now. Removed.